accel/tcg/cputlb.c

```diff
@@ -2219,9 +2219,20 @@ uint64_t cpu_ldq_le_data_ra(CPUArchState *env, target_ulong ptr,
  * TODO(am2419): Ugly hack to avoid logging memory accesses that load capability
  * components as normal memory accesses. The caller is responsible for logging.
  */
-uint64_t cpu_ldq_cap_data_ra(CPUArchState *env, target_ulong ptr,
```

Comment:
Could you split the changes that are just CHERI256/CHERI128Magic removal into a separate commit?

Reply:
No need to do anything like creating a fake intermediate state, I just mean split the files/hunks into two commits.